internal/provider/resource_gitlab_service_jira.go

```diff
@@ -182,9 +189,14 @@ func resourceGitlabServiceJiraRead(ctx context.Context, d *schema.ResourceData,
 		return diag.FromErr(err)
 	}
 
+	log.Printf("[TIMOOOO:] %+v", jiraService)
```

Comment:
@timofurrer lol, probably not intended?

Reply:
omg 😂  soo sorry. Please merge: https://github.com/gitlabhq/terraform-provider-gitlab/pull/880